puma/utils/histogram.py

```diff
@@ -156,6 +156,46 @@ def hist_w_unc(
 
     return bin_edges, hist, unc, band
 
+def filled_hist_w_unc(
```

Comment:
Do you think it would also be possible to merge this with the existing `hist_w_unc` function? I think you can just add an extra argument to that function telling it if it's the filled histogram type or not. The uncertainty calculation is the same regardless of type, the `hist_w_unc` function calls `np.histogram` which gives the bin heights and edges, whereas you start with that

Reply:
Looks like it is from the code ;)